test/runner.js

```diff
@@ -87,10 +99,42 @@ if ( program.nodeTotal > 1 ) {
 
 files.forEach( setup.addFile );
 
-mocha.addFile( path.join( __dirname, 'load-suite.js' ) );
-
-mocha.run( function( failures ) {
-	process.on( 'exit', function() {
-		process.exit( failures ); //eslint-disable-line no-process-exit
+if ( program.watch ) {
+	console.log( chalk.green( 'Watch mode enabled' ) );
+
+	let runAgain = false;
+	let runner = null;
+
+	let rerun = () => {
+		console.log( chalk.green( 'Running tests at: ' ),
+									chalk.yellow( new Date( ) ) );
```

Comment:
1) Thanks for noticing that.  I'll  fix it up
2) I'm pretty sure I was getting eslint warnings without the space.  I'll double check